apps/dashboard/app/controllers/settings_controller.rb

```diff
@@ -29,6 +29,10 @@ def settings_param
     params.require(:settings).permit(ALLOWED_SETTINGS) if params[:settings].present?
   end
 
+  def back_param
+    ActiveModel::Type::Boolean.new.cast(params[:back])
```

Comment:
By "strong params" I meant `params.permit(:back)`. I.e., permitting some while blocking everything else.

Reply:
> Shall I use this version?

Yea or `back_param[:back]`? I guess either one works for me.

Reply:
OK - Updated to use` params.permit(:back)[:back]`